dracut/30ignition/ignition-generator

```diff
@@ -35,7 +35,7 @@ add_requires() {
 # This can't be done with ConditionKernelCommandLine because that always
 # starts the unit's dependencies. We want to start networkd only on first
 # boot.
-if $(cmdline_bool 'coreos.firstboot' 0); then
+if $(cmdline_bool 'ignition.firstboot' 0) && $(cmdline_bool 'coreos.firstboot' 0); then
```

Comment:
It looks like this is requiring both arguments to be specified?

Reply:
indeed.. so we just need to drop to the one `ignition.firstboot` check and delete the other one.

Reply:
oh yea, this was a quick shot.